### PR TITLE
Added Tomiwa as a contributor

### LINKED ISSUE #1187: New Contributor - Tomiwa
Team: Marketing
{
  "title": Marketing Analyst,
  "isLead": false,
  "payPerDay": 2800,
  "createdDate": "2020-07-08T02:14:59.307535Z",
  "contributor": {
    "contributorId": U01JHJS30SC,
    "displayName": Tomiwa,
    "githubUsername": [GITHUB USERNAME],
    "profileImage": https://ca.slack-edge.com/T07PJD1FZ-U01JHJS30SC-62f5af21aed7-512,
    "slackUsername": Tomiwa
  }
}

### PR DESCRIPTION
**Pull request by Paul-Taiwo**
Fixes #1187

Account number: c6bc0d956d668ed2464684b352226f4df1f86ed213783ba82a97a2fe38252761

### CODE REVIEW

File: src/data/teams.json
Comment (jamessspanggg) [inline src/data/teams.json:772]: contributor id is incorrect

File: src/data/teams.json
Comment (jamessspanggg) [inline src/data/teams.json:775]: use this as his profile image url instead: `https://avatars1.githubusercontent.com/u/72953345?s=400&u=8a7d5931cd7e0041fc6e68683ad25aeb4e7016ce&v=4`

**Review by jamessspanggg — CHANGES_REQUESTED** [review]

File: src/data/teams.json
Comment (Paul-Taiwo) [reply src/data/teams.json:775]: Alright.

**Comment by Paul-Taiwo** [comment]
Merge this after merging [#1203]

**Review by jamessspanggg — APPROVED** [review]

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**
